media_softlet/agnostic/common/os/mos_utilities.h

```diff
@@ -3191,6 +3191,12 @@ do{                                                     \
 #define  Mos_SwizzleData(pSrc, pDst, SrcTiling, DstTiling, iHeight, iPitch, extFlags)   \
     MosUtilities::MosSwizzleData(pSrc, pDst, SrcTiling, DstTiling, iHeight, iPitch, extFlags)
 
+#ifdef _MOS_UTILITY_EXT
+#include "mos_utilities_ext_next.h"
```

Comment:
we cannot include ext header file in base headers

Reply:
Introduced a wrapper for MosSwizzleOffset to expose swizzle offset operation to other files.